common/kokkos-sampler/kp_sampler_skip.cpp

```diff
@@ -0,0 +1,239 @@
+#include <cstdio>
+#include <cstdint>
+#include <cstdlib>
+#include <cstring>
+#include <dlfcn.h>
+
+namespace KokkosTools {
+namespace Sample {
+static uint64_t uniqID           = 0;
+static uint64_t kernelCounter    = 0;
+static uint64_t kernelSampleSkip = 101;
+static int tool_verbosity;
+
+typedef void (*initFunction)(const int, const uint64_t, const uint32_t, void*);
+typedef void (*finalizeFunction)();
+typedef void (*beginFunction)(const char*, const uint32_t, uint64_t*);
+typedef void (*endFunction)(uint64_t);
+
+static initFunction initProfileLibrary         = NULL;
+static finalizeFunction finalizeProfileLibrary = NULL;
+static beginFunction beginForCallee            = NULL;
+static beginFunction beginScanCallee           = NULL;
+static beginFunction beginReduceCallee         = NULL;
+static endFunction endForCallee                = NULL;
+static endFunction endScanCallee               = NULL;
+static endFunction endReduceCallee             = NULL;
+
+extern "C" void kokkosp_init_library(const int loadSeq,
```

Comment:
all these extern "C" here need to go. The expose macros do that.

Reply:
Done.